gaia-sub-header.js

```diff
@@ -54,7 +54,7 @@ module.exports = component.register('gaia-sub-header', {
       font-weight: normal;
 
       color:
-        var(--color-zeta);
+        var(--color-epsilon);
```

Comment:
@wilsonpage That's good! So what Amy is looking for here is actually `title-color`. But I decided to follow the color convention already there. In any case, I'll submit a new PR.